dev/com.ibm.ws.org.apache.myfaces.2.3/bnd.bnd

```diff
@@ -26,35 +26,11 @@ app-resources= \
   META-INF/services/org.apache.myfaces.spi.FaceletConfigResourceProvider | \
   META-INF/services/javax.faces.application.ApplicationFactory
 
-
 Service-Component: \
-  com.ibm.ws.jsf_2_3; \
-    implementation:=com.ibm.ws.jsf.extprocessor.JSFExtensionFactory; \
-    provide:='com.ibm.wsspi.webcontainer.extension.ExtensionFactory'; \
-    serializationService=com.ibm.ws.serialization.SerializationService; \
-    classLoadingService=com.ibm.wsspi.classloading.ClassLoadingService;\
-    cdiJSFInitializerService=com.ibm.ws.jsf.shared.cdi.CDIJSFInitializer; \
-    greedy:="cdiJSFInitializerService"; \
-    optional:="cdiJSFInitializerService"; \
-    properties:="service.vendor=IBM", \
   com.ibm.ws.jsf_2_3.taglib; \
-    implementation:=com.ibm.ws.jsf.extprocessor.JSFGlobalTagLibConfig; \
+    implementation:=com.ibm.ws.jsf.taglibconfig.JSFGlobalTagLibConfig; \
```

Comment:
The package had two class files (JSFExtensionFactory.java, JSFGlobalTagLibConfig.java)  and it needed to be split (JSFGlobalTagLibConfig looks for meta files within the jar --- “META-INF/myfaces_core.tld ) . I renamed the package to be more appropriate

Reply:
Ok so just to clarify it looks as though the JSFExtensionFactory.java stay in the extprocessor package?

Reply:
`JSFExtensionFactory` is moved to the shared bundle (io.openliberty.faces.internal). The only class that remains in com.ibm.ws.org.apache.myfaces.2.3 is JSFGlobalTagLibConfig.  As mentioned above JSFGlobalTagLibConfig referes to tlds within the jar, so it needs to remain in the same bundle. 

I tried moving it to the shared bundle but, as expected, you encounter the following NPE here (as TLD can't be found): 

```
[4/21/21 9:43:47:284 EDT] 00000027 LogService-88-com.ibm.ws.jsp.jakarta                         E CWWKE0701E: bundle com.ibm.ws.jsp.jakarta:1.0.51.202104210930 (88)[com.ibm.ws.jsp.webcontainerext.JSPExtensionFactory(220)] : The setGlobalTagLibConfig method has thrown an exception java.lang.NullPointerException
	at com.ibm.ws.jsp.taglib.GlobalTagLibraryCache.loadTldFromClassloader(GlobalTagLibraryCache.java:564)
	at com.ibm.ws.jsp.taglib.GlobalTagLibraryCache.addGlobalTagLibConfig(GlobalTagLibraryCache.java:603)
	at com.ibm.ws.jsp.webcontainerext.JSPExtensionFactory.setGlobalTagLibConfig(JSPExtensionFactory.java:601)
```

Reply:
ok makes sense now to me why we're splitting the package into two packages.